requirements/requirements.txt

```diff
@@ -1,9 +1,10 @@
 colorama
 distrax @ git+https://github.com/google-deepmind/distrax  # distrax release doesn't support jax > 0.4.13
-flax==0.7.5
+flax~=0.7.5
```

Comment:
So @RuanJohn was worried that it will create a breaking change like they did previously. This `~=` means that we will stay in `0.7.x` so there shouldn't be a breaking change within here